pom.xml

```diff
@@ -734,6 +735,12 @@
                 </exclusions>
                 <scope>test</scope>
             </dependency>
+
```

Comment:
@tuichenchuxin Can you add this dependence to release note?

Reply:
done.